doc/topics/pillar/index.rst

```diff
@@ -88,6 +88,33 @@ by their ``os`` grain:
         - match: grain
         - servers
 
+Pillar definitions can also take a keyword argument ``ignore_missing``.
+When the value of ``ignore missing`` is ``True``, all errors for missing
```

Comment:
`ignore missing` --> `ignore_missing`

Reply:
That's now corrected, and a push has been made, though this new entry in the permanent record for the lack proofreading is there to stay.